scripts/build/docker/Dockerfile.template

```diff
@@ -16,6 +16,8 @@ RUN apt-get update \
     curl \
     fuse \
     git \
+    graphicsmagick \
```

Comment:
@lurch You can build ia32 on x64 Linux. The same, I guess, applies for ARMv7.

`gcc-multilib g++-multilib` is preinstalled in the electron-builder docker images.